ITISFoundation/osparc-simcore
# 🐛 Billing center: Various fixes

## What do these changes do?
* Corrects typo in WS message name causing the "add payment window" to not close after success
* "Buy credits window" also closes after successful payment
* After payment, transactions table is updated and shown

### How to test

- Open billing center in a deploy with FAKE payments (only for testing)
- add payment-method
should get 
<img width="421" alt="image" src="https://github.com/ITISFoundation/osparc-simcore/assets/32402063/b0f6a6cc-6287-49c9-9ccd-e505c1f5db52">
-
